metaseq/tasks/streaming_language_modeling.py

```diff
@@ -415,7 +415,6 @@ def get_batch_iterator(
         # thus increasing randomness. This assumes that no single document spans
         # 10 full batches, which is reasonable when batch sizes are in the
         # millions and documents are on average much smaller.
-        assert isinstance(dataset, StreamingTokenBlockDataset)
```

Comment:
This assertion should be kept.  You just need to include `StreamingSrcTgtDataset` instance, right?

Reply:
Good point. Will add this

Reply:
Done